test/ufuzz.js

```diff
@@ -754,6 +758,16 @@ function _createExpression(recurmax, noComma, stmtDepth, canThrow) {
       case p++:
         var name = getVarName();
         return name + ' && ' + name + '.' + getDotKey();
+      case p++:
```

Comment:
I think the odds of calls being generated should be bumped up by repeating `case p++:` a few times.

Reply:
Done in https://github.com/mishoo/UglifyJS2/pull/2620/commits/120083390af5aa32eeb1eb473af7f7f2cf7b1610

Reply:
I don't think https://github.com/mishoo/UglifyJS2/commit/120083390af5aa32eeb1eb473af7f7f2cf7b1610 is what we want. I meant to merely repeat the `case p++:` statement many times.

Reply:
Oh wait - I now see you doubled the odds. I'd quadruple it at least.

Reply:
Keep in mind 99% of call attempts never execute, so you need a lot of them.

Reply:
Doubled again in https://github.com/mishoo/UglifyJS2/pull/2620/commits/19db7238483537385ddf9882b9b5a65ef1483169